build.sbt

```diff
@@ -92,21 +92,23 @@ lazy val common = project.in(file("faunadb-common"))
     )
   )
 
+crossScalaVersions := scalaVersions
+
 lazy val scala = project.in(file("faunadb-scala"))
   .dependsOn(common)
   .settings(jacoco.settings)
   .settings(publishSettings : _*)
   .settings(
     name := "faunadb-scala",
-    scalaVersion := baseScalaVersion,
+    scalaVersion := scalaVersions.last,
```

Comment:
I don't see any problem, but perhaps would be nice to pin the default version into a separate variable instead of `scalaVersion.last` bc someone might add inadvertently a different version as the last element